### PR TITLE
Update some fields

### PR DESCRIPTION
**Pull request by hanahmily**
* Use general types to replace paired types
 * Update package names according to proto lint rules.

### CODE REVIEW

File: oap-server/banyandb-java-client/src/main/proto/banyandb/v1/banyandb.proto [hunk 1]
```diff
@@ -32,44 +32,18 @@ message Metadata {
...
-  bool isNull = 3;
-}
-
 // Pair is the building block of a record which is equivalent to a key-value pair.
 // In the context of Trace, it could be metadata of a trace such as service_name, service_instance, etc.
 // Besides, other fields/tags are organized in key-value pair in the underlying storage layer.
 // One should notice that the values can be a multi-value.
 message TypedPair {
+  string key = 1;
+  bool  is_null =  2;
```
Comment (wu-sheng) [inline oap-server/banyandb-java-client/src/main/proto/banyandb/v1/banyandb.proto:41]: You moved is_null out of time, how does banyandb server reponse when int type is null? What will HasInt include?

Comment (hanahmily) [reply oap-server/banyandb-java-client/src/main/proto/banyandb/v1/banyandb.proto]: has introduced a new null type which contains an underlying field type

**Review by wu-sheng — APPROVED** [review]
@lujiajing1126 Please follow this and continue on your PR.

**Comment by wu-sheng** [comment]
I will merge after CI passed. This is caused by server side change.

**Pull request merged**